pyrs990/pyrs990/_options.py

```diff
@@ -90,22 +115,23 @@
 # Index record filters #
 # -------------------- #
 
-for indexFieldName in IndexRecord._fields:
+for index_filter_field_name in IndexRecord.field_names():
```

Comment:
This is confusing to me again. Is an "index filter" a filter on an "annual record" and a "filing filter" is a filter on a "BMF record" or are they different? If they are the same, can we rename these to match so we use "annual" and "BMF" consistently everywhere?

Reply:
"index filter" is a filter that applies to either the annual index or the bmf index, but it can use either an "annual record" or a "bmf record", respectively, to access the data it needs to use for filtering, or it can even use both at once... a "filing filter" is totally different, it applies to the xml data once it has been downloaded and relies only on the "filing record"